src/Microsoft.Azure.WebJobs/StorageAttribute.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.WebJobs
+{
+    /// <summary>
+    /// Base class for all Azure Storage binding attributes.
+    /// </summary>
+    public abstract class StorageAttribute : Attribute
```

Comment:
>StorageAttribute [](start = 26, length = 16)

I don't understand the difference between StorageAccountAttribute and StorageAttribute. Similar names and structurally equivalent.  Why can't we use the existing attribute?